blocks/query-panel/category-select.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { withAPIData } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { buildTermsTree } from '../../editor/utils/terms';
```

Comment:
`buildTermsTree` is in `@wordpress/editor`, this should be a WordPress dependency.

Reply:
Moved to @wordpress/utils :)

Reply:
Good catch @mcsf, blocks library shouldn't depend on the editor. 👍